content/pytorch/concepts/tensor-operations/terms/is-tensor/is-tensor.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.is_tensor()'
+Description: 'Checks if the given object is a PyTorch tensor.'
+Subjects:
+  - 'AI'
+  - 'Data Science'
+Tags:
+  - 'AI'
+  - 'Arrays'
+  - 'Data Structures'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/computer-science'
+---
+
+In PyTorch, the **`.is_tensor()`** function checks if the given object is a PyTorch [tensor](https://www.codecademy.com/resources/docs/pytorch/tensors) and returns `True` if it is. This function is part of the `torch` module.
```

Comment:
This can be better written as:

```suggestion
In PyTorch, the **`.is_tensor()`** function checks if the given object is a PyTorch [tensor](https://www.codecademy.com/resources/docs/pytorch/tensors). It returns `True` if the object is a tensor; otherwise, it returns `False`. This function is part of the `torch` module.
```